src/components/CountUpButton.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+const CountUpButton = ({ click }) => <button onClick={click}> + </button>
```

Comment:
This is basically the same button as `CountDownButton`.
Maybe you could use `children` prop here (only if you want to add styling to this button, because otherwise it would not really make sesne to make such a small components components for wrapping `button` element)?
Also for events happening in components it is nice to prefix it with `on`, for example `onClick`, `onUpdate`, `onSomethingHappened`

```jsx
const CountButton = ({ onClick, children }) => <button {/* ...morePropsHere */} onClick={onClick}>{children}</button>

<CountButton onClick={increase}>+</CountButton>
<CountButton onClick={decrese}>-</CountButton>
```